esp/services/ws_dfu/CMakeLists.txt

```diff
@@ -101,6 +102,7 @@ target_link_libraries ( ws_dfu
          dfuXRefLib 
          dfuwu 
          roxiecommlib 
+         cryptohelper
```

Comment:
Does this still exist?

Reply:
Should just be linking jlib - otherwise this dll will also publish the functions

Reply:
agreed, should have been removed, will update.